src/clj/fluree/db/query/fql/parse.cljc

```diff
@@ -472,10 +472,16 @@
 
 (defmethod parse-pattern :graph
   [[_ graph where] vars context]
-  (let [graph* (or (parse-variable graph)
-                   graph)
-        where* (parse-where-clause where vars context)]
-    [(where/->pattern :graph [graph* where*])]))
+  (let [graph*       (or (parse-variable graph)
+                         graph)
+        graph-iri    (when (string? graph*)
```

Comment:
Is it possible to have an index graph bound to a variable? Or could it be a namespaced keyword, like `:idx-fr/DotProduct`?

Reply:
It can be a variable or a compact IRI (which would be expanded if so).